examples/01_tutorials/plot_first_level_model_details.py

```diff
@@ -197,9 +197,9 @@ def plot_contrast(first_level_model):
 # 
 # By default the drift model is a set of slow oscillating functions (Discrete Cosine transform), with a cutoff at frequency 1/128 hz.
 # We can change this cut-off, e.g. to 1/64Hz.
-# This is done by setting period_cut=64(s)
+# This is done by setting high_pass=0.01(Hz)
 
-first_level_model = FirstLevelModel(t_r, period_cut=64)
+first_level_model = FirstLevelModel(t_r, high_pass=1./64)
```

Comment:
I'd put 0.01

Reply:
We should add text to the example why we are choosing this frequency value.

0.01 is not 1./64 and the examples are tailored to a period_cut of 64, so we either stick with more acuuracy or precision, modify the examples accordingly, or give a clear explanation, even if a single line of why we are choosing this particular frequency value.

OTOH, these examples might be a good time to communicate to users that non-standard frequencies are also okay.

Reply:
We should add text to the example why we are choosing this frequency value.

0.01 is not 1./64 and the examples are tailored to a period_cut of 64, so we either stick with more acuuracy or precision, modify the examples accordingly, or give a clear explanation, even if a single line of why we are choosing this particular frequency value.

OTOH, these examples might be a good time to communicate to users that non-standard frequencies are also okay.

Reply:
I think the reason that 64s was chosen here as period_cut was merely because it is half of 128s. 128s (i.e. ~ 0.01Hz) is the default value in SPM and the standard cutoff to remove spurious low-frequency effects, like physiological signal. But if you want to contrast trials in your exp. design that are too far apart, you may want to increase the period cut; or if you have a fast-event design you may want to decrease the period cut. So, I guess there's no big reason for introducing 64s in this example. It is simply because we want to show that other cut-off frequencies are allowed by the user, depending on the characteristics of their experiments.